Dockerfile

```diff
@@ -40,6 +43,21 @@ RUN pip install --target="/install" /src
 
 # Copy minimal to main image (to keep as small as possible)
 FROM python:${PY_VERSION}-slim
+
+ENV OPENAI_TOKEN=""
+ENV DISCORD_TOKEN=""
+ENV PINECONE_TOKEN=""
+ENV DEBUG_GUILD=""
+ENV DEBUG_CHANNEL=""
+ENV ALLOWED_GUILDS=""
+ENV ADMIN_ROLES=""
+ENV DALLE_ROLES=""
+ENV GPT_ROLES=""
+ENV WELCOME_MESSAGE=""
+ENV USER_INPUT_API_KEYS=""
+ENV MODERATIONS_ALERT_CHANNEL=""
+ENV USER_KEY_DB_PATH=""
```

Comment:
Can I please ask @connorv001 why these are needed? These break just starting a docker container with the image now and it sets them to "" and breaks reading `/opt/gpt3discord/etc/environment` for the settings?

What other options do we have here? I'd like to restore being able to use the config file like I can with version <10.X of gpt3discord - What happens if these are removed from the Dockerfile?